doc/user/bgp.rst

```diff
@@ -3331,6 +3331,12 @@ Debugging
    Display Listen sockets and the vrf that created them.  Useful for debugging of when
    listen is not working and this is considered a developer debug statement.
 
+.. clicmd:: debug bgp allow-martian
+
+   Enable or disable BGP accepting martian nexthops from a peer.  Please note
+   this is not an actual debug command and this command is also being deprecated
+   and will be removed soon.  The new command is `bgp allow-martian`
```

Comment:
double ticks to highlight needed

Reply:
Better to use `` :clicmd:`bgp allow-martian` `` to create a link to the new command.

Reply:
none of the other clicmd's have a backticks.  I would hesitate to add this..... In any event I changed the code for Donatas

Reply:
Not quite - there's two different but related pieces of syntax here.

This is a command definition directive:
```rst
.. clicmd: my cool command

   blah blah blah description
```
This defines an object that can be referenced later:
```
:clicmd:`my cool command`
```
In the second case, `my cool command` will appear inline and should be automatically hyperlinked to the definition's anchor point within its page (in the HTML case).